### DISCUSSION TITLE
Clicking on the left of Slide 1, jumps to Slide 2 - is this Expected behavior?

### DISCUSSION THREAD

**Discussion by ishields** [nolimits4web/swiper#4770] [Q&A] [answered]
I've noticed that on this demo page and in my implementation of Swiper as well, clicking on the left portion of Slide 1 jumps me to Slide 2.  It doesn't always happen and depends where you click.  In my implementation each slide links to a new page, and when it happens it jumps the user to the next page vs linking.  You can see a simpler demo here though
https://swiperjs.com/demos#slides-per-view

Is this expected and is there a way to turn this off?  

![Jul-16-2021 10-22-32](https://user-images.githubusercontent.com/3834864/125962856-64b94f17-c305-42f3-b0b1-bc70d2806344.gif)

**Comment by nolimits4web** [comment, marked as answer]
Because you are doing quick short swipe. You can tweak this behavior by increasing value of `threshold` parameter

**Reply by ishields** [reply]
Thanks for your quick reply! Oh interesting.  It really just feels like a click not a swipe though.  It often happens on a single click too vs the two in my gif.  In the gif it was two independent clicks but that was just the example.  Either way I can try increasing this threshold, but this doesn't seem like desired behavior for any use case (on desktop at least).  Thanks again!

**Reply by ishields** [reply]
Awesome this works perfectly.  I think it's just because my mouse is more sensitive than others so it's sliding more easily on a click.  But adjusting the default of 0 to 5px fixed it.
